lambdas-py/tests/integration/test_steam_client_integration.py

```
@@ -0,0 +1,286 @@
+
+    # Assert
+    assert len(games) > 0, "Need at least one game to test metadata"
+
+    # Check metadata completeness
+    games_with_icons = [g for g in games if g.img_icon_url]
+    games_with_logos = [g for g in games if g.img_logo_url]
+    games_with_stats = [g for g in games if g.has_community_visible_stats]
+    played_games = [g for g in games if g.playtime_forever > 0]
+    games_with_last_played = [g for g in played_games if g.rtime_last_played]
+
+    print(f"\n📦 Metadata Completeness:")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)